### ISSUE TITLE
Copy AWSCore, AWSS3 To "Copy Bundle Resource" in Framework Project

### ISSUE THREAD

**Issue by victorzcash** [aws/aws-sdk-ios#712]
I Want to create my own framework that embed AWSCore and AWSS3 Framework (Framework Inside Framework). 

I already install cocoapods, add "$(inherited)" inside framework search path and it work fine.
But whenever I tried to put my own framework inside example project, it showed me error like this : 

> dyld: Library not loaded: @rpath/AWSS3.framework/AWSS3..........Reason: image not found

I wondered how to put AWSCore and AWSS3 to "Copy Bundle Resource", should I edit Pods-CashlezSDK-resources.sh or change code inside Podfile ?

**Comment by rohandubal** [comment]
We currently do not support embedding frameworks inside frameworks. If you compile the AWSS3 and Core frameworks as static, you should be able to do that. But that requires some research for correct build settings.

**Comment by victorzcash** [comment]
Hi @rohandubal , finally I tried to copy AWSCore & AWSS3 .Framework file to my Framework Project (Not using cocoapods) and add new "Copy File Phase" in "Build Phase" . I add AWSCore and AWSS3 to "Copy File Phase" and set "Destination" to Frameworks. last, I put new "Run Script Phase" (Build Phase) like this :  

`bash "${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/AWSCore.framework/strip-frameworks.sh"`

I delete all DerivedData then clean and build then success. It work when I tried to upload my Image to AWS, but I still don't know if .ipa that embed my own framework (aws framework inside) can be accepted in iTunes for release. Thanks
